packages/lit-element/src/lit-element.ts

```diff
@@ -216,7 +216,9 @@ export const _Φ = {
     name: string,
     value: string | null
   ) => {
-    el._$attributeToProperty(name, value);
+    // eslint-disable-next-line
+    (el as any)._$attributeToProperty(name, value);
```

Comment:
It'd be nice to cast to an internal interface...

Reply:
Yeah, I think we'll need to export some non-`@internal` types from lit-html to get ssr compiling. Reverted the change to lit-html for now.